README.md

```diff
@@ -1,2 +1,15 @@
 # app-electron
 Desktop application for Zen Audio Player
+
+Listen to YouTube videos, without the distracting visuals.
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zen-audio-player/zen-audio-player.github.io?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+[![Build Status](https://travis-ci.org/artforlife/app-electron.svg?branch=master)](https://travis-ci.org/artforlife/app-electron)
```

Comment:
Update the the 2 links on this line, replace `artforlife` with `zen-audio-player`. The badge will appear broken until we merge this PR